### DISCUSSION TITLE
Adding option for sticky cursor between multiple displays

### DISCUSSION THREAD

**Discussion by ssbarnea** [rxhanson/RectanglePro-Community#59] [Ideas]
When using multiple displays MacOS does not provide a feature to enable sticky mouse pointer, one that would prevent accidental move of cursor to the next adjacent display. See https://apple.stackexchange.com/questions/343830/stop-mouse-pointer-from-sticking-to-the-edge-of-display-in-dual-monitor-setup for details about that missing feature.

Any chance this could be implemented in RectaglePro? I often see myself going to another display instead of clicking vscode horizontal sidebar icons, just because they are near the edge.

**Comment by rxhanson** [comment]
That stack exchange link might be the wrong one? 

Either way, this might be a little too much of an edge case to bundle into Rectangle Pro. It seems more like a fit for a small open source app idea. I don't dismiss any ideas though. If people really want this then I'll be up for working it in somewhere.
